cookbooks/aws-parallelcluster-config/templates/default/directory_service/update_directory_service_password.sh.erb

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
```

Comment:
I'd add a comment describing when this tool should be used and a line with the usage (e.g. just call the script without any parameter).

Reply:
That's a very good point. Thanks